op-batcher/readme.md

```diff
@@ -166,6 +169,21 @@ curl -X POST -H "Content-Type: application/json" \
 
 The batcher tries to ensure that batches are posted at a minimum frequency specified by `MAX_CHANNEL_DURATION`. To achieve this, it caches the l1 origin of the last submitted channel, and will force close a channel if the timestamp of the l1 head moves beyond the timestamp of that l1 origin plus `MAX_CHANNEL_DURATION`. When clearing its state, e.g. following the detection of a reorg, the batcher will not clear the cached l1 origin: this way, the regular posting of batches will not be disturbed by events like reorgs.
 
+### Runtime batcher flushing
+It is possible to manually trigger the batcher to post data to the DA layer without restarting it, like so:
+```bash
+curl -X POST -H "Content-Type: application/json" \
+  --data '{"jsonrpc":"2.0","method":"admin_flushBatcher","params":[],"id":1}' \
+  http://localhost:8545
+
+# or
+cast rpc admin_flushBatcher -r http://localhost:8545
+```
+The command can either be run on the batcher host machine itself, or remotely by replacing `localhost` with the appropriate remote hostname.
+
+### Enabling the `admin` API
+This is disabled by default but can be set with `OP_BATCHER_RPC_ENABLE_ADMIN=true` or `--rpc.enable-admin=true`. It is necessary to allow the commands above to work.
```

Comment:
Hmm I'd move this above section "Runtime batcher flushing" and then write something like "in the following sections, some use the batcher's admin api. It can be enabled with ...". When we add more docs in the future this stays true.

Reply:
[3733486](https://github.com/ethereum-optimism/optimism/pull/17967/commits/3733486277179e32da8cc827a7539f73498e1f5a)